### PR TITLE
Feature/ac 24873 | Update 'Type' picklist | Ac package 1.5 | Patch 

### PR DESCRIPTION
**Pull request by AmarenderN**

### CODE REVIEW

File: 1.5.1 [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                }
+            ]
+        },
+        {
+            "taskList": [
+                {
+                    "jiraTicket": "AC-24873b",
+                    "actionType": "DeclarativeMetadata",
+                    "actionTypeName": "dummy",
+                    "description": "Update the standar field lable on the event object",
```
Comment (KilMad) [inline 1.5.1:21]: Update the standard field label on the event object*

**Review by KilMad — CHANGES_REQUESTED** [review]
Just some typos but they run!

File: 1.5.1 [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                {
+                    "jiraTicket": "ac-24873d",
+                    "description": "Path to add 1st Meeting,2nd Meeting,3rd Meeting,Delivery Meeting,Annual Review values to Type",
+                    "actionType": "PicklistValueStandard",
+                    "actionTypeName": "Type",
+                    "optionsMap": {
+                        "picklistValues": "1st Meeting,2nd Meeting,3rd Meeting,Delivery Meeting,Annual Review",
+                        "picklistOperation": "add"
+                    }
+                }
```
Comment (jduclos-ae) [inline 1.5.1:60]: Is it possible to just combine this into the 'c' task above? Or did you have a reason to keep it separate?

Comment (AmarenderN) [reply 1.5.1]: We can combine these and updated the instruction.

**Review by jduclos-ae — APPROVED** [review]

**Pull request merged**